src/Apps/Order/Routes/Shipping2/Components/SaveAndContinueButton.tsx

```diff
@@ -0,0 +1,64 @@
+import { Button, ButtonProps } from "@artsy/palette"
+import { useSaveSelectedShippingQuote } from "Apps/Order/Routes/Shipping2/Hooks/useSaveSelectedShippingQuote"
+import { useShippingContext } from "Apps/Order/Routes/Shipping2/Hooks/useShippingContext"
+import { SaveAndContinueButton_order$key } from "__generated__/SaveAndContinueButton_order.graphql"
+import { graphql, useFragment } from "react-relay"
+
+interface SaveAndContinueButtonProps {
+  order: SaveAndContinueButton_order$key
+  width?: ButtonProps["width"]
+}
+
+export const SaveAndContinueButton: React.FC<SaveAndContinueButtonProps> = ({
+  order,
+  width,
+}) => {
+  const data = useFragment(
+    graphql`
+      fragment SaveAndContinueButton_order on CommerceOrder {
+        internalID
+      }
+    `,
+    order
+  )
```

Comment:
We should use the `useFragment` hooks where we can, and start phasing out `createFragmentContainer`. 

We still can't use `usePaginationFragment` or `useReloadableFragment` because of lack of React 17 SSR suspense support, but we mostly use fragments around the app, so all good.